frontend/src/Society/Society.module.css

```diff
@@ -0,0 +1,29 @@
+.container {
+    height: 174px;
+    width: 200px;
+    border-radius: 16px;
+    box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.16), 0px 3px 6px rgba(0, 0, 0, 0.23);
+    display: flex;
+    flex-direction: column;
+}
+
+.picture {
+    max-width: 100%;
+    border-radius: 16px 16px 0px 0px;
+    height: 106px;
+    background-size: cover;
+}
+
+.socNameDiv {
+    display: flex;
+    align-items: center;
+    height: 68px;
+    border-radius: 0px 0px 16px 16px;
+}
+
+.socName {
+    font-family: Inter;
```

Comment:
consider moving `font-family` code to the global `App.css` rather than a specific class in a module where possible